src/sentry/features/manager.py

```diff
@@ -327,10 +327,13 @@ def batch_has(
         Will only accept one type of feature, either all ProjectFeatures or all
         OrganizationFeatures.
         """
+        sample_rate = 0.01
         if self._entity_handler:
-            return self._entity_handler.batch_has(
-                feature_names, actor, projects=projects, organization=organization
-            )
+            with metrics.timer("features.entity_batch_has", sample_rate=sample_rate):
```

Comment:
Doesn't need to change here since it doesn't particularly matter, but alternatively you should be able to annotate `batch_has` with `@metrics.wraps("features.entity_batch_has", sample_rate=0.01)` to keep the implementation a tiny bit cleaner